dockernet/config/relayer_config_dydx_noble.yaml

```diff
@@ -43,3 +58,19 @@ paths:
     src-channel-filter:
       rule: ""
       channel-list: []
+  noble-osmo:
```

Comment:
These are directional in the direction we expect transfers to happen, we don't care about return messages so we don't need the other directions correct?

Reply:
they're actually bidirectional!